docs/layout-props.md

```diff
@@ -905,17 +905,19 @@ Setting `paddingVertical` is like setting both of `paddingTop` and `paddingBotto
 
 ### `position`
 
-`position` in React Native is similar to regular CSS, but everything is set to `relative` by default, so `absolute` positioning is always relative to the parent.
+`position` in React Native is similar to [regular CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/position), but everything is set to `relative` by default.
 
-If you want to position a child using specific numbers of logical pixels relative to its parent, set the child to have `absolute` position.
+If you want to position a child using specific numbers of logical pixels relative to itself, set the child to have `relative` position.
```

Comment:
If `relative` is the default, can we just remove this sentence? 🤔 It feels a bit confusing to say that `relative` is the default and then to ask our users to set the position to `relative` to achieve a specific result result.

Reply:
I feel we should keep the gist of the sentence there so people know what relative does. Honestly, rereading this, I think the phrasing is just weird. The "if you want... then do this". I think I should just change this to describe what each one does without any fluff.